CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- arm64 image build support
+
+### Changed
+- remove usage of syscall() to support non-amd64 systems
```

Comment:
I think we should drop this item from the changelog - it doesn't appear it has any user impact at all. please see our [changelog guidelines](https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#changelog-guidelines) to learn more.